pkg/virt-launcher/virtwrap/device/hostdevice/sriov/pcipool_netstatus_test.go

```diff
@@ -32,9 +32,17 @@ import (
 var _ = Describe("SRIOV PCI address pool with network-pci-map", func() {
 	var emptyFileBytes []byte
 	emptyNetworkPCIMapBytes := []byte("{}")
-	networkPCIMapWithThreeNetworks := []byte(
-		`{"network1":"0000:04:02.5","network2":"0000:04:02.7","network3":"0000:04:02.8"}`)
-
+	networkPCIMapWithThreeNetworks := []byte(`{
+		"interfaces": [
+			{"network":"pod"},
```

Comment:
I do not think you need this here. It is confusing because such data is not expected to be placed in the annotation in the first place.

At the same time, the code protects against it but it is already tested directly through a dedicated test table.

Reply:
I removed the network-pod entry.